build/server/konnectors/digiposte.js

```diff
@@ -108,7 +111,8 @@ function fetchBills(requiredFields, bills, data, next) {
       connector.logger.info('App access token is ' + accessToken);
       return accessToken;
     } else {
-      return Promise.reject('Problem fetching the access token');
+      connector.logger.error('Problem fetching the access token');
+      return Promise.reject('parsing error');
```

Comment:
There is a dedicated locale key for that : https://github.com/cozy-labs/konnectors/blob/master/client/app/locales/en.json#L12